packages/connect-query/package.json

```diff
@@ -28,24 +28,24 @@
     "test": "jest"
   },
   "devDependencies": {
-    "@connectrpc/connect": "^0.13.1",
-    "@connectrpc/connect-web": "^0.13.1",
-    "@bufbuild/protobuf": "^1.3.0",
-    "@bufbuild/protoc-gen-es": "^1.3.0",
-    "@tanstack/react-query": "^4.32.6",
+    "@bufbuild/jest-environment-jsdom": "^0.1.1",
+    "@bufbuild/protobuf": "^1.3.1",
+    "@bufbuild/protoc-gen-es": "^1.3.1",
+    "@connectrpc/connect": "^0.13.2",
+    "@connectrpc/connect-web": "^0.13.2",
+    "@tanstack/react-query": "^4.33.0",
     "@testing-library/react": "^14.0.0",
-    "@types/react": "^18.2.20",
+    "@types/react": "^18.2.21",
     "@types/react-dom": "^18.2.7",
     "generated-react": "workspace:*",
-    "@bufbuild/jest-environment-jsdom": "^0.1.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "tsup": "^7.2.0",
-    "typescript": "^5.1.6"
+    "typescript": "^5.2.2"
   },
   "peerDependencies": {
-    "@connectrpc/connect": ">=0.12.0",
     "@bufbuild/protobuf": ">=1.3.0",
+    "@connectrpc/connect": ">=0.12.0",
```

Comment:
Should this be `0.13.2`?

Reply:
Since this is a peer dependency, we don't need to be that specific. Technically connect-query doesn't care that much since we're only interested in the types which don't change that often.

Reply:
If we publish a v2 of protobuf-es or connect-es, these constraints will cause problems. `^1.3.0` and `>=0.12.0 <=1` seem more safe to me.

Reply:
Updated.